spec/menu_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'menu'
+
+describe Menu do
+
+  let(:menu) { double :menu }
```

Comment:
It doesn't look like we use this variable- do we need it?

Are there other places in the tests where we can make a similar change?